pkg/liqoctl/install/provider/generic.go

```diff
@@ -80,3 +63,32 @@ func (p *GenericProvider) ValidateGenericCommandArguments(flags *flag.FlagSet) (
 
 	return nil
 }
+
+// PostValidateGenericCommandArguments validates the flags required by every install provider
+// after the specific provider validation.
+func (p *GenericProvider) PostValidateGenericCommandArguments(oldClusterName string) (err error) {
+	switch {
+	case p.ClusterName == "" && !p.GenerateClusterName && oldClusterName == "":
```

Comment:
Can you please comment more on each branch of this case to understand which situation is codified?